src/main/java/baseball/Application.java

```diff
@@ -1,7 +1,17 @@
 package baseball;
 
+import baseball.Controller.GameController;
+import baseball.View.GameView;
+import camp.nextstep.edu.missionutils.Console;
+import camp.nextstep.edu.missionutils.Randoms;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
 public class Application {
+
     public static void main(String[] args) {
-        // TODO: 프로그램 구현
+        GameController gameController = new GameController();
+        gameController.GameStart();
```

Comment:
메서드네이밍 컨벤션에 따르면 "카멜 케이스"의 규칙이 있습니다! 
```suggestion
        gameController.gameStart();
```